database1/pgQueryTests.js

```diff
@@ -0,0 +1,64 @@
+const pg = require('./pgQueryTests.js')
+const { Client } = require('pg');
+
+const client = new Client({
+  user: 'clairemelbourne',
+  database: 'similar_prod'
+});
+
+client.connect();
+
+//get request from 14000000
+const getOne = (productId, res) => {
+  const priceText = `SELECT price FROM products WHERE productid = ${productId}`;
+  client.query(priceText)
+  .then((result) => {
+    const price = result.rows[0].price
+    const productsText = `SELECT * FROM products WHERE ${getRange(productId, null)} AND price <= ${price + 50} AND price <= ${price - 50} LIMIT 8`;
+    client.query(productsText)
+    .then((result) => {
+      console.log("results of query", result.rows);
+    })
+  })
+  .catch((err) => {
+    console.error(err.stack);
+    client.end();
+  })
+}
+
+const create = (req, res) => {
+  console.log("request body", req.body);
+  const b = req.body
+  const idText = `SELECT MAX(productid) FROM products WHERE ${getRange(null, b.category)}`;
+  client.query(idText)
+  .then((result) => {
+  console.log("id", result.rows[0].max)
+  var productId = result.rows[0].max + 1;
+    var text = `INSERT INTO products (productid, name, category, price, rating, imageurl, onsale) VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING *`;
```

Comment:
Are the $s in the query the PostgreSQL way of noting that you'll be using an array of parameters? I hadn't gotten to that point yet and this will be good to know.

Reply:
Yes! As far as I have seen and functionally, $ + number is the same as ? in Cassandra. Not sure if the numbering is necessary or not but it was in all the examples I saw and works here.